values/encoding/double_encode.go

```diff
@@ -11,6 +11,10 @@ import (
 	"github.com/xichen2020/eventdb/x/proto"
 )
 
+var (
+	uint64SizeBytes = 8
```

Comment:
Hm is this needed?

Reply:
I stopped using it in another file where it was defined but it is still used in `double_encode.go` so I moved it here.

Reply:
Sounds good, can you make it a const?

Reply:
I removed the usage of this constant from another file but it is still being used in `double_encode.go` so I moved the definition here.